### PR TITLE
[TEST]Nuitka standalone test for Pyparsing PyPI package.

### PR DESCRIPTION
**Pull request by tab1tha**
The pyparsing module provides a library of classes that
client code uses to construct the grammar directly in python code.

The Pyparsing package is tested by using it to parse a string
of the form <salutation>,<greeting>! ,and then checking that
the result type is as expected( In this case,result type is a list.

Contributes to https://github.com/Nuitka/Nuitka/issues/229
Signed-off-by: tab1tha <tambetachere@gmail.com>

Thank your for contributing to Nuitka!

!! Please check that you select the **develop branch** (details see below link) !!

Before submitting a PR, please review the guidelines:
https://github.com/kayhayen/Nuitka/blob/master/CONTRIBUTING.md

### PR Checklist
- [x] Correct base branch selected? `develop` for new features and bug fixes too. If it's
      part of a hotfix, it will be moved to ``master`` during it.
- [x] All tests still pass. Check the developer manual about ``Running the Tests``. There
      are Travis tests that cover the most important things however, and you are
      welcome to rely on those, but they might not cover enough.
- [x] Ideally new features or fixed regressions ought to be covered via new tests.
- [x] Ideally new or changed features have documentation updates.

### CODE REVIEW

File: tests/standalone/PyparsingUsing.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#program to parse any greeting of the form <salutation>,<address>!
+
+from pyparsing import Word, alphas
+
+# nuitka-skip-unless-imports: Word, alphas
+
+greet = Word(alphas) + "," + Word(alphas) + "!"
+greeting = greet.parseString("Hello , World !")
+print(greeting)
```
Comment (xennygrimmato) [inline tests/standalone/PyparsingUsing.py]: Use `logging` instead of `print`

File: tests/standalone/PyparsingUsing.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+def good_parse():
+	'''Test function to check if result of pyparse is a list'''
+	assert type(greeting) == list
```
Comment (xennygrimmato) [inline tests/standalone/PyparsingUsing.py]: Can you add asserts for individual elements of the list?

File: tests/standalone/PyparsingUsing.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+from pyparsing import Word, alphas
+
+# nuitka-skip-unless-imports: Word, alphas
+
+greet = Word(alphas) + "," + Word(alphas) + "!"
+greeting = greet.parseString("Hello , World !")
+print(greeting)
+
+def good_parse():
```
Comment (xennygrimmato) [inline tests/standalone/PyparsingUsing.py]: You can structure your code in a slightly different way. You can create a class called `PyparsingGreeting`, create the `greet` object in the `__init__` method, rename the `good_parse` method to `test_greeting` and write the logic there. In the `if __name__ == '__main__'` block, you can create an instance of `PyparsingGreeting` and call the `test_greeting` method.

**Comment by tab1tha** [comment]
@tommyli3318 has been assigned to the issue that this pull request addresses. Hence I will close this pull request and halt my work on it. However, I am available to work on it and/or answer any questions pertaining to it as need be.

**Pull request closed without merging**